src/collapse/dtrace.rs

```diff
@@ -164,6 +169,21 @@ impl Folder {
         )
     }
 
+    fn demangle<'a>(&self, frame: &'a str) -> Cow<'a, str> {
+        let mut parts = frame.splitn(2, '`');
```

Comment:
Should this be `rsplitn`?

Reply:
I could change it to use `rsplitn`, but it's not clear to me why that would be better. If there is only ever one `` ` ``, it could be written either way. Is there a reason you'd prefer to change it?

Reply:
I guess it's a question of where we would expect to see another ``` if there ever was one. But you're probably right -- if we'd see it anywhere, it would be in the function name, not in the module name.